### PR TITLE
🏗🐷 Piggy-back on Github's find function for finding files

### PR DESCRIPTION
**Pull request by alanorozco**
Also made base template's script an IIFE.

### CODE REVIEW

File: build-system/app-index/template.html [hunk 1]
```diff
@@ -51,6 +51,27 @@
     line-height: 40px;
     font-size: 18px;
     margin: 0;
+    display: flex;
+  }
+  .find-icon {
```
Comment (torch2424) [inline build-system/app-index/template.html]: I know it's not the point of this PR, but while my hands are tied with other stuff, would you be able to move the CSS into it's own single file, and include similar to how we are doing `<!--examples-->` ?

Comment (torch2424) [reply build-system/app-index/template.html]: I can then go ahead and do the same in my PR once I get around to updating it 😄

File: build-system/app-index/template.html [hunk 2]
```diff
@@ -232,29 +260,34 @@ <h3 class="code" id="basepath"><!-- basepath --></h3>
...
     // TODO(alanorozco): Write this in ES2015 to be transpiled.
-    function maybeEnableExampleModeSelect() {
-      var basepath = document.getElementById('basepath');
-      if (!/^\/examples/.test(basepath.innerText)) {
-        return;
-      }
-      var form = document.querySelector('#examples-mode-form');
-      var select = document.querySelector('#examples-mode-select');
-      var links = document.querySelectorAll('.file-link');
+    (function() {
```
Comment (torch2424) [inline build-system/app-index/template.html]: Nice IIFE 😄

File: build-system/app-index/template.html (hunk 1 above)
Comment (alanorozco) [reply build-system/app-index/template.html]: Will do in a follow-up PR to prevent polluting the PR.

**Pull request merged**